libavfilter/dnn/dnn_backend_tf.c

```diff
@@ -665,26 +791,29 @@ void ff_dnn_free_model_tf(DNNModel **model)
     if (*model){
         tf_model = (TFModel *)(*model)->model;
         if (tf_model->graph){
-            TF_DeleteGraph(tf_model->graph);
+	    tf_model->tffns->TF_DeleteGraph(tf_model->graph);
         }
         if (tf_model->session){
-            TF_CloseSession(tf_model->session, tf_model->status);
-            TF_DeleteSession(tf_model->session, tf_model->status);
+            tf_model->tffns->TF_CloseSession(tf_model->session, tf_model->status);
+            tf_model->tffns->TF_DeleteSession(tf_model->session, tf_model->status);
         }
         if (tf_model->status){
-            TF_DeleteStatus(tf_model->status);
+            tf_model->tffns->TF_DeleteStatus(tf_model->status);
         }
         if (tf_model->input_tensor){
-            TF_DeleteTensor(tf_model->input_tensor);
+            tf_model->tffns->TF_DeleteTensor(tf_model->input_tensor);
         }
         if (tf_model->output_tensors) {
             for (uint32_t i = 0; i < tf_model->nb_output; ++i) {
                 if (tf_model->output_tensors[i]) {
-                    TF_DeleteTensor(tf_model->output_tensors[i]);
+                    tf_model->tffns->TF_DeleteTensor(tf_model->output_tensors[i]);
                     tf_model->output_tensors[i] = NULL;
                 }
             }
         }
+
+	av_freep(&tf_model->tffns);
+	TF_FREE_FUNC(tf_model->libtensorflow);		
```

Comment:
nit: indent mismatch here and a couple other places too.

Reply:
All indents should be good now.